boot.go

```diff
@@ -53,7 +54,8 @@ func main() {
 				}
 				fs := sys.RealFS()
 				env := sys.RealEnv()
-				pushLock := sshd.NewInMemoryRepositoryLock()
+				locktimeout, _ := strconv.Atoi(env.Get("GIT_LOCK_TIMEOUT"))
```

Comment:
can you put this into the existing [config struct](https://github.com/deis/builder/blob/master/pkg/sshd/config.go)